dds_web/static/swagger.yaml

```diff
@@ -2,22 +2,40 @@
 # yamllint disable rule:braces
 openapi: 3.0.0
 info:
-  description: |
-    Documentation for the API of the Data Delivery System (DDS) in SciLifeLab
-  version: "2.6.0"
+  description: | # html syntax is allowed
+    Documentation for the API of the Data Delivery System (DDS) in SciLifeLab Data Centre. <br>
+    This documentation is mainly intended for internal use. But developers and researchers
+     can use it at their own risk.
+    However, DC recomends to always use the client and tools provided to units and researchers. <br>
+    <br><br>
+    <b>WARNING</b>: Some endpoints are not according to the OpenAPI standard, and therefore
+    cannot be documented properly. These endpoints are marked with <u>CHECK METHOD</u> and as deprecated.<br>
+    These endpoints will be changed in the future, and will be documented properly in the <b>url</b> hosted in
+    <a href="/api/documentation/v3"> api/documentation/v3</a> <br>
+    <br>
+    <b>USE</b>: Change the server in the list below depending on the instance you are using,
+    and click on the "Authorize" button to enter your user and password. <br>
+    The endpoint <b> /user/encrypted_token </b> is used to generate an encrypted token for the user. <br>
+    Add this endpoint in the "Authorize" button and call the endpoint <b> /user/second_factor </b>
+     to authenticate with MFA. <br>
+    Lastly, save again this second token in the same place (login out the previous one). <br>
+    Now you can start using the API! <br>
```

Comment:
I tried to follow these instructions but I don't get them. 
The first two steps are fine, but 

"Add this endpoint in the "Authorize" button and call the endpoint /user/second_factor to authenticate with MFA." 

is unclear for me. And the next step, too.

Reply:
I added a better explanation